### PR TITLE
Fix browser launch

### PR DESCRIPTION
**Pull request by swmarks**
Removed canLaunchUrl check

### CODE REVIEW

**Comment by swmarks** [comment]
Additional context:

https://github.com/liftoff-app/liftoff/pull/35#issuecomment-1612762157

**Review by zachatrocity — APPROVED** [review]
LGTM

File: lib/url_launcher.dart [hunk 1]
```diff
@@ -114,7 +114,8 @@ Future<bool> launchLink({
   required BuildContext context,
 }) async {
   final uri = Uri.tryParse(link);
-  if (uri != null && await ul.canLaunchUrl(uri)) {
+  //if (uri != null && await ul.canLaunchUrl(uri)) {
```
[the remark this answers is not in the document]
Comment (swmarks) [reply lib/url_launcher.dart]: Yeah, good catch, done

**Comment by zachatrocity** [comment]
these are failing due to timeout likely instance related.

**Pull request merged**